### PR TITLE
Restructure distributed tracing

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):
4.10+

Issue:

https://issues.redhat.com/browse/TRACING-3288

Link to docs preview:

https://61948--docspreview.netlify.app/

QE review:
- [x] QE has approved this change.

Additional information:

There is no documentation content change in this PR, only structure was changed. 

This PR splits `Distributed tracing/Distributed tracing installation` into:
* `Distributed tracing/Distributed tracing platform - Jaeger` 
* `Distributed tracing/Distributed tracing data collection - OpenTelemetry`

The spit is necessary because we want to introduce `Distributed tracing/Distributed tracing platform - Tempo` and it will be better to have documentation per component. In the future we will probably move the `Distributed tracing data collection - OpenTelemetry` into the root level as it might become a separate product. 

Each of these components will host following sub articles:
* instaling
* configuring 
* updating
* removing

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -3576,19 +3576,24 @@ Topics:
   Topics:
   - Name: Distributed tracing architecture
```
Comment (IshwarKanse) [inline _topic_maps/_topic_map.yml:3577]: This could be in the same level as [Distributed tracing release notes](https://61948--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html) Currently we have to click on the Distributed tracing architecture dropdown arrow and then again select Distributed tracing architecture to view the page.

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -3576,19 +3576,24 @@ Topics:
...
-    File: distr-tracing-installing
-  - Name: Configuring the distributed tracing platform
-    File: distr-tracing-deploying-jaeger
-  - Name: Configuring distributed tracing data collection
-    File: distr-tracing-deploying-otel
-  - Name: Upgrading distributed tracing
-    File: distr-tracing-updating
-  - Name: Removing distributed tracing
-    File: distr-tracing-removing
+- Name: Distributed tracing platform - Jaeger
```
Comment (IshwarKanse) [inline _topic_maps/_topic_map.yml:3579]: Should we use camel case for the topic names ? Like Distributed Tracing Platform - Jaeger and Distributed Tracing Data Collection - OpenTelemetry.

Comment (pavolloffay) [reply _topic_maps/_topic_map.yml:3579]: The official name of the product is `Red Hat OpenShift distributed tracing platform` (lower case after `OpenShift`), but here I guess we can use capitalization as we want. 

I will leave this to docs team to decide

File: _topic_maps/_topic_map.yml (hunk 1 above)
Comment (pavolloffay) [reply _topic_maps/_topic_map.yml:3577]: agree, but I consider this out of the scope of this work.  I will open a separate PR to fix this

File: distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="dist-tracing-otel-installing"]
+= Installing distributed tracing data collection OpenTelemetry
+include::_attributes/common-attributes.adoc[]
+:context: install-distributed-tracing-otel
+
+toc::[]
+
```
Comment (max-cx) [inline distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc:8]: [pavolloffay](https://github.com/pavolloffay), I'm not acquainted with this product. I noticed that this page is missing the following paragraphs that are in https://61948--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing.html. Is this a deliberate omission? Or are these paragraphs not relevant to the content on this page?
image

Comment (pavolloffay) [reply distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc:8]: Correct, and this is a mistake. The OpenTelemetry does not integrate with service mesh in any form. The service mesh docs do not contain anything about the OTEL operator.

File: service_mesh/v2x/ossm-reference-jaeger.adoc
Comment (max-cx) [inline service_mesh/v2x/ossm-reference-jaeger.adoc:1]: BTW, just curious: what's your plan regarding the content in https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-reference-jaeger.html?

File: modules/distr-tracing-accessing-jaeger-console.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ////
 Module included in the following assemblies:
-* distr_tracing/distr_tracing_install/distr-tracing-deploying-jaeger.adoc
 * distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc
```
Comment (max-cx) [inline modules/distr-tracing-accessing-jaeger-console.adoc:3]: Could you please double-check this reference. It didn't seem up to date to me:
```
* distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc
```

File: _topic_maps/_topic_map.yml (hunk 2 above)
Comment (max-cx) [inline _topic_maps/_topic_map.yml:3579]: I noticed that the section about [the CLI tools uses parentheses for the tool name in the nav entry](https://docs.openshift.com/container-platform/4.13/cli_reference/index.html), so you could adopt the same convention:
```suggestion
- Name: Distributed tracing platform (Jaeger)
```

File: _topic_maps/_topic_map.yml [hunk 5]
```diff
@@ -3576,19 +3576,24 @@ Topics:
...
+  Topics:
+    - Name: Installing Jaeger
+      File: distr-tracing-jaeger-installing
+    - Name: Configuring Jaeger
+      File: distr-tracing-jaeger-configuring
+    - Name: Updating Jaeger
+      File: distr-tracing-jaeger-updating
+    - Name: Removing Jaeger
+      File: distr-tracing-jaeger-removing
+- Name: Distributed tracing data collection - OpenTelemetry
```
Comment (max-cx) [inline _topic_maps/_topic_map.yml:3590]: I noticed that the section about [the CLI tools uses parentheses for the tool name in the nav entry](https://docs.openshift.com/container-platform/4.13/cli_reference/index.html), so you could adopt the same convention:
```suggestion
- Name: Distributed tracing data collection (OpenTelemetry)
```

**Review by max-cx — APPROVED** [review]
@pavolloffay, I left a few more comments here. Please review them. Pre-approving this PR.

File: distr_tracing/distr_tracing_otel/distr-tracing-otel-configuring.adoc [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
 include::_attributes/common-attributes.adoc[]
-:context: deploying-distr-tracing-data-collection
+:context: deploying-distr-tracing-otel
 
 toc::[]
 
-The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements. 
+The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELName} resources. You can either install the default configuration or modify the file to better suit your business requirements.
 
 include::modules/distr-tracing-config-otel-collector.adoc[leveloffset=+1]
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply distr_tracing/distr_tracing_otel/distr-tracing-otel-configuring.adoc:11]: done

File: service_mesh/v2x/ossm-reference-jaeger.adoc
Comment (pavolloffay) [reply service_mesh/v2x/ossm-reference-jaeger.adoc:1]: That is a question for the service mesh team. I think in the past they wanted to have Jaeger installation docs directly in their docs, however, they are recently trying to decouple from 3rd party tools, so they will maybe just send user to jaeger docs.

**Review by max-cx — APPROVED** [review]

**Review by IshwarKanse — APPROVED** [review]

**Comment by max-cx** [comment]
@pavolloffay, fyi, I've merged your changes from this PR into PR https://github.com/openshift/openshift-docs/pull/62410.

**Comment by pavolloffay** [comment]
It will be done as part of https://github.com/openshift/openshift-docs/pull/62410
